components/esp_rom/include/esp32/rom/uart.h

```diff
@@ -122,7 +122,7 @@ typedef enum {
 } UartFlowCtrl;
 
 typedef enum {
-    EMPTY,
+    EMPTY_,
```

Comment:
Don't get why add this underline, and why it is at the end instead of the beginning like C3 and S2.

Reply:
This is due to collision with other 'EMPTY', defined as `#define EMPTY`. Underline placement is a typo with no meaning.